Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ruby:2.6.2-alpine
+
+RUN apk add --no-cache --update \
+  build-base \
+  git \
+  htop \
+  linux-headers \
+  nodejs \
+  postgresql-client \
+  postgresql-dev \
+  sqlite-dev \
+  tzdata \
+  vim \
```

Comment:
`vim` isn't needed in the image. It should be removed. It can always be added if someone is debugging in the image.

Reply:
done 8448f71577c8c6793a4a09374837776563b8d141